docs/modules/ROOT/pages/general-info/technical-preview.adoc

```diff
@@ -1,11 +1,11 @@
 [[overview]]
 = Features in Technical Preview
 
-Some features that are new in the latest version of Payara Server or Payara Micro are released in _technical preview_. This means that the feature is ready to be used and also commercially supported, but we recommend trying out the feature extensively in a test environment before using in production. 
+Some features that are new in the latest version of Payara Server Community or Payara Micro Community are released in _technical preview_. This means that the feature can already be used but is not yet final or fully tested.  If you want to use such a feature, we recommend trying out the feature extensively in a test environment before using it in production. 
```

Comment:
Mmm, not sure if we can say "trying out the feature extensively in a test environment before using it in production.", since we aren't recommending the use of Community edition in production. I'd remove this sentence altogether